index.html

```diff
@@ -204,6 +204,7 @@ <h2 id="footer-heading">CONTACT US</h2>
      </form>
```

Comment:
- [ ] [OPTIONAL] Your button should `<button id="footer-button">Send</button> `  but your button text is not like this.
Please make changes!